config/routes/detsis.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+namespace :detsis do
+  get '/', to: 'dashboard#index'
```

Comment:
Veya

```suggestion
  root to: "dashboard#index"
```

Reply:
bunu biliyordum ve kişisel olarak tercih edeceğim kullanım da bu olurdu ancak konvansiyonel olarak projede böyle kullanılmamış. o yüzden tercih etmedim.

Reply:
eğer @omu/backend okay'lerse tüm route'ları buna göre refaktör edebiliriz.

Reply:
Refaktörleyebiliriz.

Reply:
`root` kullanırsak `detsis_path` yerine `detsis_root_path` kullanmamız gerekecek. `root` yerine `get` kullanırsak daha şık olur.

Reply:
ben bu şekilde bakmıyorum. `detsis` tek başına bir isim uzayı. bu yüzden `root`u olması normal. tıpkı uygulama `root`u gibi. sanırım Rails geliştiricileri de böyle düşünerek bu imkanı vermişler. diğer projelere baktığımda da (GitLab, OpenProject) bu kullanımı görüyorum. bence çok maliyetli değil.

Reply:
eksepsiyonel durumlar olabilir ama. her senaryo için iddialı konuşamam. bazı hallerde `get`e geçirilen argümanlar ile `root to:` geçirilen argümanlar farklılık gösterebilir, hatta `get` kullanmaya zorlayabilir